### PR TITLE
SAAS-595 add side menu

### PR DESCRIPTION
**Pull request by nicolalamacchia**
image

### CODE REVIEW

**Review by fabio-silva — APPROVED** [review]

File: src/components/IconLink/IconLink.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { TestContainer } from 'components/TestContainer';
+import { IconLink } from './IconLink';
+
+const pixel = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=';
+const testAltText = 'test';
+const testText = 'Test';
+
+describe('IconLink', () => {
+  it('renders the specified icon and text', async () => {
+    render(<TestContainer><IconLink alt={testAltText} to="/test" icon={pixel}>{testText}</IconLink></TestContainer>);
```
Comment (atymchuk) [inline src/components/IconLink/IconLink.test.tsx:12]: Can we have the usual JSX formatting in the render call?

**Review by atymchuk — DISMISSED** [review]

**Comment by atymchuk** [comment]
@nicolalamacchia Pls check the failing unit test

**Comment by atymchuk** [comment]
Any reason we skip running e2e tests?

**Review by fabio-silva — DISMISSED** [review]

File: src/components/IconLink/IconLink.test.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply src/components/IconLink/IconLink.test.tsx]: Done

**Comment by nicolalamacchia** [comment]
> Any reason we skip running e2e tests?

They should be running 🤔

**Review by atymchuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**